### PR TITLE
Can't hide mouse cursor #6180

### PR DESCRIPTION
**Pull request by FrancescoBonizzi**

### CODE REVIEW

File: MonoGame.Framework/Platform/WindowsUniversal/InputEvents.cs [hunk 1]
```diff
@@ -35,6 +35,22 @@ internal class KeyChar
...
+        private CoreIndependentInputSource _coreIndependentInputSource;
+
+        /// <summary>
+        /// Sets the cursor of <see cref="CoreIndependentInputSource"/> thread
+        /// </summary>
+        public CoreCursor CoreCursor
+        {
+            set
+            {
+                _coreIndependentInputSource.Dispatcher.TryRunAsync(
```
Comment (harry-cpp) [inline MonoGame.Framework/Platform/WindowsUniversal/InputEvents.cs]: You are never setting `_coreIndependentInputSource`, its always gonna be null?

Comment (FrancescoBonizzi) [reply MonoGame.Framework/Platform/WindowsUniversal/InputEvents.cs]: Hi Harry! I'm gonna check to answer you.

Comment (FrancescoBonizzi) [reply MonoGame.Framework/Platform/WindowsUniversal/InputEvents.cs]: I made a fix Harry. Sorry for this, I made a mistake with the first pull request, then after creating the new one I lost a piece.

**Comment by harry-cpp** [comment]
Rebase from upstream/develop.

**Comment by FrancescoBonizzi** [comment]
@harry-cpp "Rebase from upstream/develop.". Sorry, I didn't understand if I should do it or if you are doing it.

**Comment by harry-cpp** [comment]
> @harry-cpp "Rebase from upstream/develop.". Sorry, I didn't understand if I should do it or if you are doing it.

You need to do it so the build passes, right now its failing. You can do either a pull or rebase.

**Comment by FrancescoBonizzi** [comment]
I think I did it:

image

**Comment by harry-cpp** [comment]
When you are in your repo, in git terminal, type in:
```
git fetch upstream
git rebase upstream/develop
git push -u origin Issue6180 --force
```

**Comment by FrancescoBonizzi** [comment]
image

**Comment by FrancescoBonizzi** [comment]
Sorry for the inconvenience, now I understood.

**Comment by Jjagg** [comment]
Thanks @FrancescoBonizzi!
Sorry for the hold up, we're going through old PR's and this one's good :)

**Pull request merged**